docs/source/docs/level-up/intermediate_skills/index.rst

```diff
@@ -10,8 +10,9 @@ Intermediate Skills
    06_data_comparison
    07_data_merge
    08_data_validate
-   09_data_exploration
-   10_data_generation
+   09_data_filtering
+   10_data_exploration
+   11_data_generation
```

Comment:
It is bothering but you have to increase the lesson number of https://github.com/openvinotoolkit/datumaro/tree/b44bea6a094c0e72b63821d8c9f6e547b8820262/docs/source/docs/level-up/advanced_skills as well.

Reply:
Fix in [`1104c4d`](https://github.com/openvinotoolkit/datumaro/commit/1104c4d3e8e39592e6e7993aa2ab494b45acea64). If there is anything which I missed, please let me know.